.github/workflows/gemini-review.yml

```
@@ -0,0 +1,138 @@
+        with:
+          node-version: 20
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Install dependencies
+        run: |
+          pnpm add -Dw @google/generative-ai js-yaml
+
+      - name: Get git diff for PR
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)